en/integrations/plugin-packs/procedures/cloud-azure-storage-acr.md

```diff
@@ -0,0 +1,285 @@
+---
+id: cloud-azure-storage-acr
+title: Azure Container Registry
+---
+
+## Overview
+
+Azure Container Registry is a managed, private Docker registry service based on the open-source Docker Registry 2.0. Create and maintain Azure
+Container Registries to store and manage your private Docker container images and related artifacts.
+
+The Centreon Pack *Azure Container Registry* can rely on Azure API or Azure CLI to collect the metrics related to the *ACR* service.
+
+## Pack Assets
+
+### Monitored Objects
+
+Available indicators and metrics:
+* Azure Container Registries
+     * Storage-usage
+
+### Discovery rules
+
+The Centreon Pack *Azure Container Registry* includes a Host Discovery *provider* to automatically discover the Azure instances of a given
+subscription and add them to the Centreon configuration.
+This provider is named **Microsoft Azure Container Registry**:
+
+![image](../../../assets/integrations/plugin-packs/procedures/cloud-azure-storage-acr-provider.png)
+
+> This discovery feature is only compatible with the 'api' custom mode. 'azcli' is not supported.
+More information about the Host Discovery module is available in the Centreon documentation:
+[Host Discovery](../../../monitoring/discovery/hosts-discovery.html)
+
+### Collected metrics
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Storage-usage-->
+
+| Metric Name                                            | Description  | Unit |
+|:-------------------------------------------------------|:-------------|:-----|
+| *instance*#azure.containerregistry.storage.used.bytes  | Storage used | B    |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+To get data from Azure Services, following methods are available:
+    * Azure API ('api') 
+    * Azure CLI ('azcli')
+
+Centreon recommends to use the API instead of the CLI for the following reasons:
+    * API is much more efficient by avoiding CLI binary execution
+    * API supports application authentication while CLI does not (yet)
+
```

Comment:
Please add something to say that the Monitoring Reader Azure role is required for the app.